_component_design/chips.md

```diff
@@ -9,28 +9,28 @@ preview-image: preview-images/chip.svg
 resource: true
 status: stable
 last-modified: 2017-11-16
```

Comment:
Ensure `last-modified` is updated whenever a document is edited. 

YYYY-MM-DD format is used.

Reply:
Got it, made those changes.